### ISSUE TITLE
HTTPS everywhere

### ISSUE THREAD

**Issue by intenscia** [DBolical/Home#5]
Need to follow best practices

**Comment by theunbeholden** [comment]
I agree, its odd for this site to be the only one I frequent that I'm not guaranteed encryption/security.
